.github/workflows/lint.yaml

```diff
@@ -45,4 +45,4 @@ jobs:
             echo dev-tools=${REGISTRY}/${IMAGE}:${VERSION} >> "$GITHUB_OUTPUT"
           fi
       - run: docker run --rm -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} module-swapper
-      - run: docker run --rm -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
+      - run: docker run --rm -e DISABLE_TFLINT=1 -v ${{ github.workspace }}:/workspace ${{ steps.variables.outputs.dev-tools }} /usr/local/bin/test_lint.sh
```

Comment:
This is because i added symlinks on fleetscope, the tflint can't read the links - same case happens in foundation codebase:

```
├── 3-fleetscope
│   ├── common.auto.example.tfvars
│   ├── development.auto.example.tfvars
│   ├── envs
│   │   ├── development
│   │   │   ├── backend.tf
│   │   │   ├── common.auto.tfvars -> ../../common.auto.tfvars
│   │   │   ├── development.auto.tfvars -> ../../development.auto.tfvars
│   │   │   ├── main.tf
│   │   │   ├── README.md
│   │   │   ├── variables.tf
│   │   │   └── versions.tf
│   │   ├── nonproduction
│   │   │   ├── backend.tf
│   │   │   ├── common.auto.tfvars -> ../../common.auto.tfvars
│   │   │   ├── main.tf
│   │   │   ├── nonproduction.auto.tfvars -> ../../nonproduction.auto.tfvars
│   │   │   ├── README.md
│   │   │   ├── variables.tf
│   │   │   └── versions.tf
│   │   └── production
│   │       ├── backend.tf
│   │       ├── common.auto.tfvars -> ../../common.auto.tfvars
│   │       ├── main.tf
│   │       ├── production.auto.tfvars -> ../../production.auto.tfvars
│   │       ├── README.md
│   │       ├── variables.tf
│   │       └── versions.tf

```

Reply:
I'm curious, we are currently using symlinks in 2-multitenant and it isn't an issue: https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/2-multitenant/envs/development/terraform.tfvars